[BUG] (v3.2.0) `_cat/nodes` API shows negative values in cpu stats

### Describe the bug

I ran into the below issue in 3.2.0 version that cpu usages could be negative and the same value (-1)

```
$ curl "https://dev-opensearch-test-250821-os.osp01.abc.com/_cat/nodes?v" -ku admin:test

ip             heap.percent ram.percent cpu load_1m load_5m load_15m node.role node.roles      cluster_manager name
10.251.175.203           38          60  -1    0.44    0.41     0.49 m         cluster_manager *               dev-opensearch-test-250821-master-1
10.251.117.144           20          60  -1    0.58    0.54     0.60 m         cluster_manager -               dev-opensearch-test-250821-master-0
10.251.225.190           67          90  -1    1.79    1.14     1.03 di        data,ingest     -               dev-opensearch-test-250821-data-0
10.251.143.196           24          60  -1    0.40    0.68     0.61 m         cluster_manager -               dev-opensearch-test-250821-master-2
```

### Related component

Clients

### To Reproduce

1. deploy opensearch cluster using 3.2.0 version
2. After your cluster runs successfully, `curl "https://dev-opensearch-test-250821-os.osp01.abc.com/_cat/nodes?v" -ku admin:test`
3. It also happens wrongly when you try `_cluster/stats` API

### Expected behavior

.

### Additional Details

**Plugins**
I just added the prometheus-exporter plugin by building new Docker image
```
FROM harbor.abc.com/library/opensearchproject/opensearch:3.2.0

COPY prometheus-exporter-3.2.0.0.zip /tmp/
RUN ./bin/opensearch-plugin install file:/tmp/prometheus-exporter-3.2.0.0.zip
```

**Host/Environment (please complete the following information):**
 - OS: All hosts are worker nodes of kubernetes v1.31.4 and based on RHEL 8.10 (4.18.0 kernel)
 - Version: OpenSearch 3.2.0

**Additional context**
- Air-gapped environment but I believe it doesn't matter with this issue.

Hi @reta ,
You were absolutely right about the JDK behavior.

1. Log Findings
I checked the logs in /usr/share/opensearch/logs/ and there are no exceptions referencing /proc/. The only error present is an unrelated ClassNotFoundException for org.opensearch.performanceanalyzer.PerformanceAnalyzerApp.

```
Error: Could not find or load main class org.opensearch.performanceanalyzer.PerformanceAnalyzerApp
Caused by: java.lang.ClassNotFoundException: org.opensearch.performanceanalyzer.PerformanceAnalyzerApp
```

2. Swapped JVM Test
I updated my test class to switch the order of the calls then waiting 2 seconds and execute again, and the results perfectly inverted just as you suspected, confirming the JDK-8338904 behavior:

<img width="1089" height="715" alt="Image" src="https://github.com/user-attachments/assets/a438a28c-def4-44f9-95cf-0272489bf4b0" />

3. Next steps

While this explains why a 0.0 might appear depending on the order of invocation, the JVM test is successfully returning valid floats. It never returns -1.

However, the OpenSearch API `/_nodes/_local/stats/os` is still reporting `"percent" : -1`. Since the underlying JVM OperatingSystemMXBean is clearly able to read the CPU load without failing, is it possible that OpenSearch's OS stats probe is explicitly converting 0.0 (or extremely low values) to -1, or perhaps catching a different silent exception when it polls these metrics?

Here is the complete output for `/_nodes/_local/stats/os`   [os-stats.json](https://github.com/user-attachments/files/28728893/os-stats.json)

--- a/server/src/main/resources/org/opensearch/bootstrap/security.policy
+++ b/server/src/main/resources/org/opensearch/bootstrap/security.policy
@@ -246,6 +246,8 @@ grant {
   permission java.io.FilePermission "/sys/fs/cgroup/cpu", "read";
   permission java.io.FilePermission "/sys/fs/cgroup/cpu/-", "read";
   permission java.io.FilePermission "/sys/fs/cgroup/cpu.max", "read";
+  // cgroup v2: read by the JVM's container-aware OperatingSystemMXBean.getCpuLoad() (usage_usec/nr_periods)
+  permission java.io.FilePermission "/sys/fs/cgroup/cpu.stat", "read";
   permission java.io.FilePermission "/sys/fs/cgroup/cpu.weight", "read";
   permission java.io.FilePermission "/sys/fs/cgroup/cpuset.cpus", "read";
   permission java.io.FilePermission "/sys/fs/cgroup/cpuset.cpus.effective", "read";

